mycroft/tts/espeak_tts.py

```diff
@@ -32,8 +32,32 @@ def get_tts(self, sentence, wav_file):
         Returns:
             tuple ((str) file location, None)
         """
-        subprocess.call(['espeak', '-v', self.lang + '+' + self.voice,
-                         '-w', wav_file, sentence])
+
+        # Create Argument String for Espeak
+        arguments = ['espeak', '-v', self.lang + '+' + self.voice]
+        argument = self.config.get("amplitude")
+        if argument:
+            arguments.append("-a "+argument)
+
+        argument = self.config.get("gap")
```

Comment:
The rest of this module is using single quotes - would be great to keep that consistency.

Reply:
Also thanks for the Feedback :)
Updated it.